### DISCUSSION TITLE
getting error when i am clone the Repository

### DISCUSSION THREAD

**Discussion by naveenkumar481** [community/community#77922] [New to GitHub]
![image](https://github.com/community/community/assets/82015066/a2168b41-c42c-4efc-93ea-bcb0375f677c)

**Comment by resslr** [comment]
You need to enable port 8087, check if there are any apps currently using this port by closing all other apps, use the CMD (I assume you are on windows) app and then run git clone there

**Comment by naveenkumar481** [comment]
i close all cmd again getting same

**Comment by resslr** [comment]
Try installing something called WSL

**Comment by naveenkumar481** [comment]
again getting same error
